### PR TITLE
Fix: Close to Collateral Fee collection

### PR DESCRIPTION
**Pull request by zerotucks**
**Description**
Updated how we collect fees when closing to collateral. Fee collection now occurs in the normal manner where the swap contract handles fee collection.

We still use a similar mechanism to bump the amount of collateral being swapped (to allow for fee collection). 

**Changes**
- Updated logic governing how fees are collected to use Swap contract
- Removed sendToken action from CloseToColl operation where the difference between the o/s debt and swap result are sent to the fee wallet
- Updated fee calculations to use feeResolver utility

**How to Test**
- `yarn hardhat test test/aave/CloseToCollateral.test.ts`

**Additional work**
*Deploy updated operations using multi sig

### CODE REVIEW

File: test/aave/CloseToCollateral.test.ts [hunk 1]
```diff
@@ -258,23 +252,22 @@ describe('Close AAVEv2 Position to collateral', () => {
...
+    // The precision of the two comparison amounts has been lowered to 8.
+    // This is because in the time between us getting the current position
+    // And executing the transaction, the collateral accrues interest.
+    // When we withdraw the collateral with the MAX_UINT flag we get a different collateral amount
+    // To the amount we had earlier after querying the position
+    // By lowering the sensitivity of the comparison we can avoid this issue
     expectToBeEqual(
-      amountFromWei(expectedBalance).toFixed(16),
-      amountFromWei(userCollateralBalanceAfterTx).toFixed(16),
+      amountFromWei(expectedBalance).toFixed(9),
```
Comment (peculiarity) [inline test/aave/CloseToCollateral.test.ts]: It's lowered to 8 in the comment but you put 9 in the `toFixed`

File: packages/oasis-actions/src/helpers/swap/acceptedFeeToken.ts [hunk 2]
```diff
@@ -30,6 +30,10 @@ export function acceptedFeeToken({ fromToken, toToken }: Props): 'sourceToken' |
     return fallbackTokenType
   }
 
+  if (fromTokenAcceptedIndex === -1) {
```
Comment (peculiarity) [inline packages/oasis-actions/src/helpers/swap/acceptedFeeToken.ts]: Can you elaborate on this one ? What's the idea behind it.

File: test/aave/CloseToCollateral.test.ts (hunk 1 above)
Comment (zerotucks) [reply test/aave/CloseToCollateral.test.ts]: Done

File: packages/oasis-actions/src/helpers/swap/acceptedFeeToken.ts (hunk 2 above)
Comment (zerotucks) [reply packages/oasis-actions/src/helpers/swap/acceptedFeeToken.ts]: Have added some comments and improved variable names to try and make clearer

File: test/aave/CloseToCollateral.test.ts (hunk 1 above)
Comment (zerotucks) [reply test/aave/CloseToCollateral.test.ts]: Corrected 👍

**Review by peculiarity — APPROVED** [review]

**Pull request merged**